pkgs/by-name/gg/gg-jj/package.nix

```
@@ -0,0 +1,95 @@
+{
+  lib,
+  stdenv,
+
+  rustPlatform,
+  fetchFromGitHub,
+  fetchNpmDeps,
+  yq,
+  moreutils,
+
+  cargo-tauri,
+  cargo,
+  rustc,
+  nodejs,
+  npmHooks,
+  pkg-config,
+  wrapGAppsHook3,
+
+  openssl,
+  webkitgtk_4_1,
+
+  versionCheckHook,
+  nix-update-script,
+}:
+stdenv.mkDerivation (finalAttrs: {
+  pname = "gg";
+  version = "0.23.0";
+
+  src = fetchFromGitHub {
+    owner = "gulbanana";
+    repo = "gg";
+    rev = "refs/tags/v${finalAttrs.version}";
+    hash = "sha256-iQxPJgMxBtyindkNdQkehwPf7ZgWCI09PToqs2y1Hfw=";
+  };
+
+  cargoRoot = "src-tauri";
+  buildAndTestSubdir = "src-tauri";
+
+  # FIXME: Switch back to cargoHash when https://github.com/NixOS/nixpkgs/issues/356811 is fixed
+  cargoDeps = rustPlatform.fetchCargoTarball {
+    inherit (finalAttrs) pname version src;
+    sourceRoot = "${finalAttrs.src.name}/${finalAttrs.cargoRoot}";
+    hash = "sha256-Lr/0GkWHvfDy/leRLxisuTzGPZYFo2beHq9UCl6XlDo=";
+
+    nativeBuildInputs = [
+      yq
+      moreutils
+    ];
+
+    # Work around https://github.com/rust-lang/cargo/issues/10801
+    # See https://discourse.nixos.org/t/rust-tauri-v2-error-no-matching-package-found/56751/4
+    preBuild = ''
+      tomlq -t '.dependencies.tauri.features += ["native-tls"]' Cargo.toml | sponge Cargo.toml
+    '';
+  };
+
+  npmDeps = fetchNpmDeps {
+    inherit (finalAttrs) pname version src;
+    hash = "sha256-SMz1ohPSF5tvf2d3is4PXhnjHG9hHuS5NYmHbe46HaU=";
+  };
+
+  nativeBuildInputs = [
+    cargo-tauri.hook
+    rustPlatform.cargoSetupHook
+    cargo
+    rustc
+    nodejs
+    npmHooks.npmConfigHook
+    pkg-config
+    wrapGAppsHook3
+  ];
+
+  buildInputs =
+    [ openssl ]
```

```suggestion
    [ openssl ]
      ++ lib.optional stdenv.hostPlatform.isDarwin apple-sdk_11
```
Fixes the [build error](https://logs.ofborg.org/?key=nixos/nixpkgs.360354&attempt_id=4702ae85-716f-45c8-b1d3-2c1188593fed) on x86_64-darwin